packages/client-api-schema/src/types.ts

```diff
@@ -181,6 +175,10 @@ export type GetBudgetResponse = JsonRpcResponse<SiteBudget | {}>;
 
 export type ApproveBudgetAndFundRequest = JsonRpcRequest<'ApproveBudgetAndFund', BudgetRequest>;
 export type ApproveBudgetAndFundResponse = JsonRpcResponse<SiteBudget>;
+
+export type CloseAndWithdrawParams = {site: string; player: Participant; hub: Participant};
```

Comment:
Should  `site` just be pulled from the domain the json-rpc request originates from?

Reply:
Probably. It seems like we should be able to figure out which `player` we are too?

Reply:
I've created #1258 to make sure our API applies this across the board.